roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -75,8 +74,7 @@ struct State {
 
 Check DB for recorded votes for non concluded disputes we have not yet
 recorded a local statement for.
-For all of those send `DisputeParticipationMessage::Participate` message to
-dispute participation subsystem.
+For all of those initiate dispute participation.
```

Comment:
Dispute participation was deleted. But the changeset here doesn't describe what 'initiating dispute participation' even means.